agent/bench-scripts/postprocess/fio-postprocess

```diff
@@ -147,6 +138,37 @@ if ( @cpu_busy ) {
 	$resource{'cpu_busy'} = \@cpu_busy;
 }
 
+# get the fio benchmark parameters from the fio-result.txt (which is mostly json data)
+my %benchmark_params;
+my $fio_results_scalar = get_json($dir . "/fio-result.txt");
+my %fio_results = %$fio_results_scalar;
+for my $param (keys $fio_results{"global options"} ) {
+	if (not $param =~/^write_\w+_log$/) {
+		$benchmark_params{$param} = $fio_results{'global options'}{$param};
+	}
+}
+if ($fio_results{"client_stats"}) {
+	for my $client (@{ $fio_results{"client_stats"} }) {
+		for my $param (keys $$client{"job options"} ) {
+			if (not $param =~/^write_\w+_log$/) {
+				$benchmark_params{$param} = $$client{"job options"}{$param};
+			}
+		}
+	}
+} else {
+	for my $client (@{ $fio_results{"client_stats"} }) {
```

Comment:
What is different between this block and the one above?

Reply:
The difference is where the stats are stored.  One is for when using --client with fio, and one is for running a local fio job.  The "client_stats" being the difference.

FWIW, I plan on removing the use of --clients and running fio via ssh like we do with uperf, so this code will get smaller.  I actually would prefer I do both of these changes with the same pull request, so I am thinking of just cancelling this PR.

Reply:
Actually, I do have a mistake there, extra reference to client_stats in the second section.  I will fix...